doxygen.cmake

```diff
@@ -184,6 +184,28 @@ MACRO(_SETUP_PROJECT_DOCUMENTATION_FINALIZE)
       SET(DOXYGEN_HEADER_NAME "header.html")
     ENDIF()
 
+    # Find doxytag files
+    # To ignore this list of tag files, add to doc/Doxyfile.extra.in
+    # TAGFILES =
+    STRING(REPLACE "," ";" PKG_REQUIRES "${_PKG_CONFIG_REQUIRES}")
+    FOREACH(PKG_CONFIG_STRING ${PKG_REQUIRES})
+      # Get the name of the package in the same format as in pkg-config.cmake.
+      STRING(REGEX MATCH "[^<>= ]+" LIBRARY_NAME "${PKG_CONFIG_STRING}")
+      STRING(REGEX REPLACE "[^a-zA-Z0-9]" "_" PREFIX "${LIBRARY_NAME}")
+      STRING(TOUPPER "${PREFIX}" "PREFIX")
```

Comment:
To simplify future maintenance, I'd rather have a macro/function that does this transformation in `pkg-config.cmake` and invoke it here.